client/Packages/com.beamable/Common/Runtime/Modules/Shop/ListingRef.cs

```diff
@@ -17,5 +17,15 @@ namespace Beamable.Common.Shop
 	/// </summary>
 	[System.Serializable]
 	[Agnostic]
-	public class ListingRef : ContentRef<ListingContent> { }
+	public class ListingRef : ContentRef<ListingContent>
+	{
+		public ListingRef()
+		{
+
+		}
+
+		public ListingRef(string id) : base(id)
+		{
+		}

```

Comment:
It is bugging me that there is no blank line in this one like there is in all the rest 🥴
As such, I demand that you keep it in, as a reminder that syntax is independent of whitespace